cmd/fsi.go

```diff
@@ -34,8 +34,9 @@ func NewFSICommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	}
 
 	unlink := &cobra.Command{
-		Use:   "unlink",
-		Short: "unlink a .qri-ref",
+		Use:   "unlink [DATASET]",
+		Short: "unlink a dataset from a directory on disk",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
Well now I have more questions. I used `MaximumNArgs` instead of `ExactArgs` here because I was thinking this would operate on the current directory or the `use` dataset if none was specified, but somehow totally missed that the first thing we do when parsing the arg values is to ensure there is at least one arg: https://github.com/qri-io/qri/blob/924157c253bf571ec9df92b0fd63e316e67bc6ad/cmd/fsi.go#L62-L65

…I left it as `MaximumNArgs` so that code can print a nicer message than `ExactArgs` would. BUT now I’m wondering if this is as expected (you must always specify a dataset name, in which case I need to fix the usage string) or if it *should* be falling back to the current directory or whatever was specified with `use` (in which case our parsing needs to change). Any thoughts here?

Reply:
hmmm, What do you think?

I'm of the opinion that the FSI commands should _all_ be sensitive to `pwd`. After all, that's kinda the point of FSI. It'd be nice to state "all FSI commands default to the current linked directory if one exists".

cc @dustmop, who is generally the maintainer of FSI related things these days

Reply:
If I were just going on what I expected/thought seemed reasonable, I’d say the first arg to `qri fsi unlink` should be optional and it should default to the working directory, like other qri commands. But then the same probably goes for `qri fsi link` — the second arg (the directory to link to) should be optional and defaults to the current working dir. But now this change has gotten pretty complex. 😜

Reply:
Good point. I vote we merge this & follow up with a PR that does a more comprehensive audit of default args 😄.